Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -360,9 +362,9 @@ BEGIN
     LTEXT           "Topic",IDC_STATIC_CHTOPIC,12,38,89,8
     EDITTEXT        IDC_EDIT_CHANTOPIC,105,38,105,40,ES_MULTILINE | ES_AUTOVSCROLL | ES_WANTRETURN | WS_VSCROLL
     LTEXT           "Password",IDC_STATIC_CHPASSWORD,12,86,91,8
-    EDITTEXT        IDC_EDIT_CHANPASSWORD,105,83,78,14,ES_AUTOHSCROLL
+    EDITTEXT        IDC_EDIT_CHANPASSWORD,105,83,78,14,ES_PASSWORD | ES_AUTOHSCROLL

```

Comment:
The issue with hidden passwords have been discussed a bit before. Most people prefer channel passwords to be visible. Also it allows administrators to see them.